packages/driver/src/cy/commands/sessions/index.ts

```diff
@@ -21,7 +22,18 @@ type SessionData = Cypress.Commands.Session.SessionData
  */
 
 export default function (Commands, Cypress, cy) {
+  // @ts-ignore
+  Object.values(Cypress.state('activeSessions') || {}).forEach((sessionData: ServerSessionData) => {
+    if (sessionData.cacheAcrossSpecs) {
```

Comment:
any benefit here to making this type safe?
```suggestion
    if (sessionData?.cacheAcrossSpecs) {
```

Reply:
not a bad idea so we don't just bork out.